src/builder.rs

```diff
@@ -347,6 +351,62 @@ impl<'ctx> Builder<'ctx> {
         PointerValue::new(value)
     }
 
+    /// Alignment arguments are specified in bytes, and should always be a power of 2.
+    ///
+    /// The final argument should be a pointer-sized integer.
+    ///
+    /// [`TargetData::ptr_sized_int_type_in_context`](https://thedan64.github.io/inkwell/inkwell/targets/struct.TargetData.html#method.ptr_sized_int_type_in_context) will get you one of those.
+    #[llvm_versions(8.0..=latest)]
+    pub fn build_memcpy(
+        &self,
+        dest: PointerValue<'ctx>,
+        dest_align_bytes: u32,
+        src: PointerValue<'ctx>,
+        src_align_bytes: u32,
+        size: IntValue<'ctx>,
+    ) -> Result<PointerValue<'ctx>, &'static str> {
+        let value = unsafe {
+            LLVMBuildMemCpy(
+                self.builder,
+                dest.as_value_ref(),
+                dest_align_bytes,
+                src.as_value_ref(),
+                src_align_bytes,
+                size.as_value_ref(),
+            )
+        };
+
+        Ok(PointerValue::new(value))
+    }
+
+    /// Alignment arguments are specified in bytes, and should always be a power of 2.
+    ///
+    /// The final argument should be a pointer-sized integer.
+    ///
+    /// [`TargetData::ptr_sized_int_type_in_context`](https://thedan64.github.io/inkwell/inkwell/targets/struct.TargetData.html#method.ptr_sized_int_type_in_context) will get you one of those.
+    #[llvm_versions(8.0..=latest)]
+    pub fn build_memmove(
+        &self,
+        dest: PointerValue<'ctx>,
+        dest_align_bytes: u32,
+        src: PointerValue<'ctx>,
+        src_align_bytes: u32,
+        size: IntValue<'ctx>,
+    ) -> Result<PointerValue<'ctx>, &'static str> {
```

Comment:
Same comment as above, this function cannot fail for what I read for the moment.

Reply:
Changed both in 9a263ebabd3f9f3b7d59ff001cd55d076be366fe